src/parser/input.h

```diff
@@ -59,7 +59,7 @@ struct ParseInput {
   std::optional<uint8_t> takeU8();
   std::optional<double> takeF64();
   std::optional<float> takeF32();
-  std::optional<std::string_view> takeString();
+  std::optional<std::string> takeString();
```

Comment:
Why did this change?

Reply:
The string view this method returned was dangling whenever the string contained an escape sequence, so I had to change it to return an owned string to avoid ownership problems.